tests/features/login.feature

```diff
@@ -0,0 +1,29 @@
+Feature: login
+        As an admin
+        I want to log in to the website admin panel
+        So that I can manage my products
+
+        Background:
+                Given user 'admin' has navigated to the admin login page
+
+
+        Scenario: login with valid credentials
```

Comment:
Indent your code accordingly.
```suggestion
  As an admin
  I want to log in to the website admin panel
  So that I can manage my products

  Background:
    Given user 'admin' has navigated to the admin login page


  Scenario: login with valid credentials
```